docs/pages/textarea.js

```diff
@@ -36,219 +53,270 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
-        description={`
-    A \`TextArea\` with an anchor ref to a Popover component
-  `}
-        defaultCode={`
-function TextAreaPopoverExample() {
-  const [open, setOpen] = React.useState(false);
-  const anchorRef = React.useRef();
-  return (
-    <Box marginBottom={12}>
-      <TextArea
-        ref={anchorRef}
-        label="Focus the TextArea to show the Popover"
-        id="my-example"
-        onChange={() => {}}
-        onBlur={() => setOpen(false)}
-        onFocus={() => setOpen(true)}
-      />
-      {open && (
-        <Popover
-          anchor={anchorRef.current}
-          idealDirection="down"
-          onDismiss={() => setOpen(false)}
-          shouldFocus={false}
-          size="md"
         >
-          <Box padding={3}>
-            <Text weight="bold">Example with Popover</Text>
-          </Box>
-        </Popover>
-      )}
-    </Box>
-  );
-}
-`}
-      />
-      <Example
-        id="tagsExample"
-        name="Example: Tags"
-        description={`
+          <MainSection.Card
+            defaultCode={`
+  function Example(props) {
+    const [value, setValue] = React.useState('')
+    return (
+      <Box width="100%">
+        <TextArea
+          id="witherror"
+          onChange={({value}) => setValue(value)}
+          errorMessage={!value ? "This field can't be blank!" : null}
+          placeholder="Write something about yourself..."
+          label="About me"
+          value={value}
+        />
+      </Box>
+    );
+  }
+      `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With a ref"
+          description={`
+          A \`TextArea\` with an anchor ref to a Popover component
+        `}
+        >
+          <MainSection.Card
+            defaultCode={`
+          function TextAreaPopoverExample() {
+            const [open, setOpen] = React.useState(false);
+            const anchorRef = React.useRef();
+            return (
+              <Box marginBottom={12} width="100%">
+                <TextArea
+                  ref={anchorRef}
+                  label="Focus the TextArea to show the Popover"
+                  id="my-example"
+                  onChange={() => {}}
+                  onBlur={() => setOpen(false)}
+                  onFocus={() => setOpen(true)}
+                />
+                {open && (
+                  <Popover
+                    anchor={anchorRef.current}
+                    idealDirection="down"
+                    onDismiss={() => setOpen(false)}
+                    shouldFocus={false}
+                    size="md"
+                  >
+                    <Box padding={3}>
+                      <Text weight="bold">Example with Popover</Text>
+                    </Box>
+                  </Popover>
+                )}
+              </Box>
+            );
+          }
+          `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With Tags"
```

Comment:
IMO this is referring to tags as a concept and not the Tag component specifically, so sentence case would make sense